### DISCUSSION TITLE
Animated cube

### DISCUSSION THREAD

**Discussion by FredJ21** [makerforgetech/modular-biped#65] [General]
Hello !

In the project presentation video from August 5, 2023, we can see the initial concept: “Animated cube”.

--> https://youtu.be/Nqp4vuDWgpw?si=QYSkL6mZ8p4-UVr8&t=118

Is this cube license free ?
Where does this design come from? Are there other designs of the same type ? 🤔
Can I use this wonderful cube in my projects?
😁

(my projects are license free and not commercial)

**Comment by danic85** [comment]
Hi @FredJ21! Good question - and not something I have been asked before.

That design was laser cut, I have been digging through my files for you but unfortunately I can't find this file anywhere. The service I used to print it has also deleted my account after so many years, so I think we're out of luck there.

Feel free to use the idea if you want to recreate it yourself. I just used an open source vector graphics software to design it, and it wasn't particularly complex.

Thanks!

**Reply by FredJ21** [reply]
OK, Thanks 

No problem for the laser cut or printer 3D files , 
It's so strange to make a friendly cube , so quickly and so easily 😁


Thanks for sharing your project
